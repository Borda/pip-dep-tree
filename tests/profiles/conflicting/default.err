Warning!!! Possibly conflicting dependencies found:
* Jinja2==2.11.1
 - MarkupSafe [required: >=0.23, installed: 0.22]
------------------------------------------------------------------------
