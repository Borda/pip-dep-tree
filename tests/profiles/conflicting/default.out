Flask==0.10.1
  - itsdangerous [required: >=0.21, installed: 0.24]
  - Jinja2 [required: >=2.4, installed: 2.11.1]
    - MarkupSafe [required: >=0.23, installed: 0.22]
  - Werkzeug [required: >=0.7, installed: 0.11.2]
pipdeptree==2.0.0b1
  - pip [required: >=6.0.0, installed: 20.0.2]
setuptools==46.1.3
wheel==0.34.2
